spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,117 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :request do
+  describe '#create' do
+    subject { post(register_path, params: params) }
+
+    let(:name) { 'Foo Bar' }
+    let(:email) { 'foobar@mail.com' }
+    let(:password) { 'securepassword123' }
+    let(:role) { 'user' }
+    let(:params) do
+      {
+        name: name,
+        email: email,
+        password: password,
+        password_confirmation: password,
+        role: role
+      }
+    end
```

Comment:
Acho que vc pode usar assim já que, pelo que eu vi, vc não usa essas outras variáveis em lugar nenhum

```suggestion
    let(:params) do
      {
        name: 'Foo Bar',
        email: 'foobar@mail.com',
        password: 'securepassword123',
        password_confirmation: password,
        role: 'user'
      }
    end
```

Reply:
`name` uso na linha [78](https://github.com/Leoruwer/expenses-manager-api/pull/7/files#diff-5f44c1c580dbc94788e40f1aecf6f711215f30ce917214dc8d25765105cc4a37R78)
`password` uso na linha [91](https://github.com/Leoruwer/expenses-manager-api/pull/7/files#diff-5f44c1c580dbc94788e40f1aecf6f711215f30ce917214dc8d25765105cc4a37R91)
`email` uso na linha [52](https://github.com/Leoruwer/expenses-manager-api/pull/7/files#diff-5f44c1c580dbc94788e40f1aecf6f711215f30ce917214dc8d25765105cc4a37R52)
`role` uso na linha [37](https://github.com/Leoruwer/expenses-manager-api/pull/7/files#diff-5f44c1c580dbc94788e40f1aecf6f711215f30ce917214dc8d25765105cc4a37R37)